importer/Uploader.py

```diff
@@ -59,6 +59,15 @@ def make_quantity_item(self, quantity, repo):
             unit = None
         return pywikibot.WbQuantity(amount=value, unit=unit, site=repo)
 
+    def make_monolingual_item(self, quantity, repo):
+        """
+        Create claim for a monolingual text.
+
+        quantity: {'text': 'Q11573', 'lang': 6.85}
```

Comment:
Maybe rename the variable and rewrite the example with actually expected values (strings, not qid and float)? I guess it's copypasted from make_quantity_item, took me a moment to parse.

Reply:
Yupp a whole lot of copypasta